### ISSUE TITLE
Regarding  blackbox Accessibility testing 

### ISSUE THREAD

**Issue by svshubhm** [davetrux/accessibilitysetup#1]
Hello Dave Sir,
Can you please guide me , how can I scan different apps  for accessibility checks  whose source is not available  like black box testing?
Is it possible using Espresso?
Waiting for your reply.
Thanking you in Anticipation!

**Comment by davetrux** [comment]
You can use the [Accessibility Scanner app](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor). This is a manual process though, not something you can automate.

In Espresso you can add `AccessibilityChecks.enable()` to the tests. I do not know how well that works without the code though, I have not tried it.

**Comment by svshubhm** [comment]
Thank you Sir for your reply,
I tried Accessibility scanner, its a quite impressive app.
I am trying  to build  a similar  app but with respect to Indian government  guidelines for mobile accessibility.
Any help will be appreciated.
